modules/manage/pages/manage-xdcr/filter-xdcr-replication.adoc

```diff
@@ -172,6 +172,9 @@ Conversely, if this option is not checked, the TTL _is_ made part of the replica
 
 For information on TTL and expiration, see xref:learn:buckets-memory-and-storage/expiration.adoc[Expiration].
 
+Note that the replication of deletions is _not_ prevented by the specifying of a filter, formed with regular and other filtering expressions: to ensure that document-deletions are _not_ replicated, the appropriate deletion-filter checkboxes must be checked.
```

Comment:
replication of deletions or expirations are not prevented...

Reply:
This is a little problematic, in that in the UI, we refer to all three filters as 'Deletion Filters'; whereas here, we are interested in deletion proper and also expiration. It gets quite wordy referring to the two separately. So, in attempt to simplify, I've added a couple of paragraphs at the start, explaining 'deletion filters' and equating expiration with deletion. Let me know if you think this works (and indeed is accurate). If not, I'll go through and specify each in detail.